doc/source/user/absolute_beginners.rst

```diff
@@ -864,11 +864,11 @@ result of multiplying the elements together, ``std`` to get the standard
 deviation, and more. ::
 
   >>> data.max()
```

Comment:
On line 838, in the section about Broadcasting, data is redefined `data = np.array([1.0, 2.0])`. So the text is correct, the image is wrong. I think the easiest thing would be to redefine `data` here to match the image (which will restore it to the same values as in the "Indexing and slicing" section):

```suggestion
  >>> data = np.array([1, 2, 3])
  
  >>> data.max()
```

Then at some point someone could regenerate the image in the "Broadcasting" section and change the text so that the value of `data` remains consistent throughout this page. When they do that, it would be nice to add a `.. note` how they created the image.